api/v1alpha1/clusterextension_types.go

```diff
@@ -510,6 +510,11 @@ type ClusterExtensionInstallStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Installed Bundle",type=string,JSONPath=`.status.install.bundle.name`
+//+kubebuilder:printcolumn:name=Version,type=string,JSONPath=`.status.install.bundle.version`
+//+kubebuilder:printcolumn:name="Installed",type=string,JSONPath=`.status.conditions[?(@.type=='Installed')].status`
+//+kubebuilder:printcolumn:name="Progressing",type=string,JSONPath=`.status.conditions[?(@.type=='Progressing')].status`
```

Comment:
Add `Progressing` condition reason as well?

Reply:
With that, it can get pretty wide, as it's a non-constrained string/ That might be something they need to use the `-o yaml` for.